src/main/webapp/app/tasks/status-form/status-form.component.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { TaskService } from '../task.service';
+
+import { StatusFormComponent } from './status-form.component';
+
+describe('StatusFormComponent', () => {
+  let component: StatusFormComponent;
+  let fixture: ComponentFixture<StatusFormComponent>;
+  let taskService: jasmine.SpyObj<TaskService>;
+
+  beforeEach(async () => {
+    taskService = jasmine.createSpyObj('TaskService', ['getAll']);
+    await TestBed.configureTestingModule({
+      declarations: [ StatusFormComponent ],
+      providers: [{
+        provide: 'TaskService',
+        useValue: taskService
+      }]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(StatusFormComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
tests missing

Reply:
I thought I had exceeded the allotted time for the task too much so I couldn't develop it. Besides, I need to improve my test skills.